### PR TITLE
More small fixes

### PR DESCRIPTION
**Pull request by tcanabrava**

### CODE REVIEW

File: src/ui/MainWindow.cc [hunk 1]
```diff
@@ -368,30 +370,22 @@ void MainWindow::_showDockWidget(const QString& name, bool show)
...
-        widget = new Linecharts(widgetName, _mapName2Action[widgetName], mavlinkDecoder, this);
-    } else if (widgetName == _uasInfoViewDockWidgetName) {
-        QGCTabbedInfoView* pInfoView = new QGCTabbedInfoView(widgetName, _mapName2Action[widgetName], this);
-        pInfoView->addSource(mavlinkDecoder);
-        widget = pInfoView;
-    } else {
-        qWarning() << "Attempt to create unknown Inner Dock Widget" << widgetName;
-	return;
+    switch(action->data().toInt()) {
+    case MAVLINK_INSPECTOR: widget = new QGCMAVLinkInspector(widgetName, action, MAVLinkProtocol::instance(),this); break;
```
Comment (DonLakeFlyer) [inline src/ui/MainWindow.cc]: The switch coding style is to have "case" indented from "switch". Also the actual case code should be on separate lines indented from the case statement.

Comment (tcanabrava) [inline src/ui/MainWindow.cc]: sorry - too used to kernel coding style. :)
fixed. do you wanna a second PR ?

Comment (DonLakeFlyer) [inline src/ui/MainWindow.cc]: Just send another commit on this one

Comment (tcanabrava) [inline src/ui/MainWindow.cc]: okay. commit is ready, if you wanna merge this, the other will be done in a minute.

**Comment by DonLakeFlyer** [comment]
Are you rebased to master? The UT problem should be fixed on master.

**Comment by tcanabrava** [comment]
closing because of a new PR.
